### DISCUSSION TITLE
DBus api documentation

### DISCUSSION THREAD

**Discussion by daniel-nicolai-hansten** [openthread/openthread#8728] [Q&A] [answered]
Hi

Is there some documentation for the DBus API?
We are coming from Wpantund and used this when we developed our solution https://github.com/openthread/wpantund/blob/master/doc/wpan-dbus-protocol.md

Or is there a well documented source file I can try to get the DBUS interface from?.

Sorry for my bad english, not my native language.
-Daniel

**Comment by wgtdkp** [comment, marked as answer]
This xml file defines the dbus APIs https://github.com/openthread/ot-br-posix/blob/main/src/dbus/server/introspect.xml
